constructor/nsis/_nsis.py

```diff
@@ -229,6 +230,18 @@ def rm_regkeys():
             pass
 
 
+def win_del(dirname):
+    try:
+        out = check_output('DEL /F/Q/S {} > NUL'.format(dirname), shell=True, stderr=STDOUT, cwd=dirname)
+    except CalledProcessError as e:
+            if e.returncode != 5:
+                print("Removing folder {} the fast way failed. Output was: {}".format(dirname))
```

Comment:
Hi, isn't there an argument missing?
```suggestion
                print("Removing folder {} the fast way failed. Output was: {}".format(dirname, out))
```